garage/envs/mujoco/maze/maze_env.py

```diff
@@ -282,6 +282,7 @@ def _find_goal_range(self):  # this only finds one goal!
                     maxy = i * size_scaling + size_scaling \
                            * 0.5 - self._init_torso_y
                     return minx, maxx, miny, maxy
+        return None
```

Comment:
Hmm an exception is probably in order here before you actually return (but I'm assuming pylint wants you to return something, which is fine).

Reply:
👍